src/sdk/pynni/nni/evolution_tuner/evolution_tuner.py

```diff
@@ -116,39 +92,20 @@ def json2paramater(x, is_rand, random_state, oldy=None, Rand=False, name=NodeTyp
         else:
             y = dict()
             for key in x.keys():
-                y[key] = json2paramater(x[key], is_rand, random_state, oldy[key]
+                y[key] = json2parameter(x[key], is_rand, random_state, oldy[key]
                                         if oldy != None else None, Rand, name + "[%s]" % str(key))
     elif isinstance(x, list):
         y = list()
         for i, x_i in enumerate(x):
-            y.append(json2paramater(x_i, is_rand, random_state, oldy[i]
+            if isinstance(x_i, dict):
+                if NodeType.NAME not in x_i.keys():
```

Comment:
if you want to check '_name', you have to provide additional doc content to tell users that if they want to use evolution tuner they have to add '_name' in their search space.

Reply:
Added in the doc.